### ISSUE TITLE
Is `':minimal` a diff-style option?

### ISSUE THREAD

**Issue by gdw2** [lambdaisland/kaocha#429]
This implies that the `diff-style` options are `:none` and `:minimal`:

https://github.com/lambdaisland/kaocha/blob/main/src/kaocha/report.clj#L260-L267

But the docs say `:none` and `:deep`:

https://github.com/lambdaisland/kaocha/blob/e9da79061a9773b795939473d9b304f4b367dce7/src/kaocha/runner.clj#L43

Or I'm completely mistaken!

**Comment by oxalorg** [comment]
Yes `:minimize` was recently added for a more compact diff. Looks like we should update the cli runner help doc.

**Comment by joannecheng** [comment]
Fixed in #465
